tests/cli/workflow/nf_analysis/test_cli_run.py

```diff
@@ -205,50 +244,74 @@ def test_resume_with_id(
     assert "tw runs relaunch" in caplog.text
 
 
-def test_resume_without_id(
+@pytest.mark.parametrize(
+    "context,case_id",
+    [
+        ("raredisease_context", "raredisease_case_id"),
+        ("rnafusion_context", "rnafusion_case_id"),
+        ("taxprofiler_context", "taxprofiler_case_id"),
+    ],
+)
+def test_resume_without_id_error(
     cli_runner: CliRunner,
-    rnafusion_context: CGConfig,
+    context: CGConfig,
     caplog: LogCaptureFixture,
-    rnafusion_case_id: str,
+    case_id: str,
     mock_config,
-    rnafusion_mock_analysis_finish,
+    request,
 ):
-    """Test resume command without providing NF-Tower ID when a Trailblazer Tower config file from a previous run
-    exist."""
+    """Test resume command without providing NF-Tower ID and without existing Trailblazer Tower config file."""
     caplog.set_level(logging.INFO)
+    context = request.getfixturevalue(context)
 
-    # GIVEN case-id
+    # GIVEN a case-id
+    case_id: str = request.getfixturevalue(case_id)
 
     # GIVEN a mocked config
 
     # WHEN dry running with dry specified
-    result = cli_runner.invoke(run, [rnafusion_case_id, "--dry-run"], obj=rnafusion_context)
-
-    # THEN command should execute successfully
-    assert result.exit_code == EXIT_SUCCESS
+    cli_runner.invoke(run, [case_id, "--dry-run"], obj=context)
 
-    # THEN command should use tower for relaunch
-    assert "Workflow will be resumed from run" in caplog.text
-    assert "tw runs relaunch" in caplog.text
+    # THEN command should raise error
+    assert "Could not resume analysis: No NF-Tower ID found for case" in caplog.text
+    pytest.raises(FileNotFoundError)
 
 
-def test_resume_without_id_error(
+@pytest.mark.parametrize(
+    "context,case_id",
+    [
+        ("raredisease_context", "raredisease_case_id"),
+        ("rnafusion_context", "rnafusion_case_id"),
+        ("taxprofiler_context", "taxprofiler_case_id"),
+    ],
+)
+def test_with_config_use_nextflow(
     cli_runner: CliRunner,
-    rnafusion_context: CGConfig,
+    context: CGConfig,
     caplog: LogCaptureFixture,
-    rnafusion_case_id: str,
+    case_id: str,
     mock_config,
+    request,
 ):
-    """Test resume command without providing NF-Tower ID and without existing Trailblazer Tower config file."""
+    """Test command with case_id and config file using nextflow."""
     caplog.set_level(logging.INFO)
+    context = request.getfixturevalue(context)
 
-    # GIVEN case-id
+    # GIVEN a case-id
+    case_id: str = request.getfixturevalue(case_id)
 
     # GIVEN a mocked config
 
     # WHEN dry running with dry specified
```

Comment:
No dry run specified

Reply:
I do see it line 306

Reply:
Resolving for now, feel free to reopen is that was not the test you meant